crates/sui-json-rpc/src/api.rs

```diff
@@ -279,16 +281,28 @@ pub trait RpcFullNodeReadApi {
         /// the version of the queried object. If None, default to the latest known version
         version: SequenceNumber,
     ) -> RpcResult<GetPastObjectDataResponse>;
+}
+
+#[open_rpc(namespace = "sui", tag = "Governance Read API")]
+#[rpc(server, client, namespace = "sui")]
+pub trait GovernanceReadApi {
```

Comment:
Do you mind filing a GitHub issue like https://github.com/MystenLabs/sui/issues/7183 (with the same set of tags) after this is merged?